pkg/utils/mydumper.go

```diff
@@ -81,3 +87,33 @@ func ParseMetaData(filename string) (mysql.Position, string, error) {
 
 	return pos, gtid, nil
 }
+
+func readFollowingGTIDs(br *bufio.Reader, flavor string) (string, error) {
+	var following strings.Builder
+	for {
+		line, err := br.ReadString('\n')
+		if err == io.EOF {
+			return following.String(), nil // return the previous, not including the last line.
+		} else if err != nil {
+			return "", err
+		}
+
+		line = strings.TrimSpace(line[:len(line)-1])
```

Comment:
use `line[:len(line)-1]` to trim `\n`? seems that `TrimSpace` can trim `\n`

Reply:
you are right. addressed in https://github.com/pingcap/dm/pull/679/commits/fd00838494d25dd5cbe8d4acd76995fbf54c3c90.